src/contexts/theme-context/theme-context.js

```diff
@@ -0,0 +1,28 @@
+import { createContext, useContext, useState, useEffect } from "react";
+
+const ThemeContext = createContext("");
```

Comment:
suggestion: since you are putting some fallback values, so these will be the default values of the properties which you pass in the value object(see line 20) and not the state default value